### ISSUE TITLE
Validator incorrectly mandating width and height in SVG

### ISSUE THREAD

**Issue by KDean-Dolphin** [validator/validator#1988]
### Summary

In SVG, `width` and `height` attributes are used to size [`rect`](https://www.w3.org/TR/SVG2/shapes.html#RectElement), [`svg`](https://www.w3.org/TR/SVG2/struct.html#SVGElement), [`image`](https://www.w3.org/TR/SVG2/embedded.html#ImageElement), and [`foreignObject`](https://www.w3.org/TR/SVG2/embedded.html#ForeignObjectElement) elements.

The [documentation](https://www.w3.org/TR/SVG2/geometry.html#Sizing) doesn't go into detail about the `width` and `height` attribute values, referring the reader instead to the CSS specification:

> See the CSS 2.1 specification for the definitions of [width](https://www.w3.org/TR/CSS21/visudet.html#propdef-width) and [height](https://www.w3.org/TR/CSS21/visudet.html#propdef-height).

Each definition specifies an initial value of `auto` which, according to [CSS rules](https://www.w3.org/TR/2011/REC-CSS2-20110607/cascade.html#specified-value), makes it the default.

Although the [SVG 1.1 DTD](https://www.w3.org/TR/SVG11/svgdtd.html#DTD.1.22) mandates these attributes, SVG 2 [doesn't have a DTD at all](https://www.w3.org/TR/2015/WD-SVG2-20150709/intro.html#Namespace), so the use of the SVG namespace to load the SVG 1.1 DTD (if that's what validator is doing) is incorrect.

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
<svg><rect/><svg>
```

### Checker error messages

```shell
error: Element “rect” is missing required attribute “width”.
error: Element “rect” is missing required attribute “height”.
```

### Steps to reproduce

1.
2.
3.

### Expected behavior

Accepts absent `width` and `height`.

### Actual behavior

Errors on absent `width` and `height`.
